ci-operator/config/stackrox/collector/stackrox-collector-mauro-refactor-stackrox-collector-config.yaml

```diff
@@ -266,32 +270,11 @@ resources:
       memory: 2Gi
 tests:
 - as: lint
-  steps:
-    test:
-    - as: clang-format
-      commands: make -C collector check
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-    - as: shfmt-check
-      commands: make shfmt-check
-      from: src
-      resources:
```

Comment:
Why do we remove the resources? is it just that it's not so important for these tests?

Reply:
(same question for the other tests )

Reply:
When using the short, non-steps version of tests, there is no `resources` entry that can be added to the element, I believe you need to specify it in the global `resources` element instead, but to be honest I think it makes no difference.